gear/csv_center_splitter/src/python/csv_center_splitter_app/run.py

```diff
@@ -29,14 +30,22 @@ def __init__(self,
                  adcid_key: str,
                  target_project: str,
                  staging_project_id: Optional[str] = None,
+                 include: Optional[List[str]] = None,
+                 exclude: Optional[List[str]] = None,
                  delimiter: str = ',',
                  local_run: bool = False):
         super().__init__(client=client)
 
+        if (include and exclude and set(include) & set(exclude)):
```

Comment:
Please use the intersection method instead of the `&` operator.

The python documentation suggests that use of the operator can be error prone. Not necessarily as you have used it, but if someone changes something in the future